README.md

```diff
@@ -187,11 +187,12 @@ func main() {
 <img src="img/custom.svg" alt="custom" width=500/>
 </p>
 
-More examples can be found in the `examples/` folder.
+More examples can be found in the `examples/` folder which you can explore yourself.
+Feel free to add more examples and submit a pr for the [issue](https://github.com/Delta456/box-cli-maker/issues/37)!

```

Comment:
No need to mention about this issue because it is assigned solely for the ASoC program.